R/config.R

```diff
@@ -0,0 +1,114 @@
+#' Riverfly species and water quality reading type mappings
+#' A set of named lists that map internal column names to human-readable names for Riverfly species and water quality reading types.
+#' These mappings are used throughout the package for data processing and visualization.
+#' @format A list of named lists:
+#' \describe{
+#' \item{riverfly_spp_bw}{A named list mapping Riverfly species internal column names to human-readable names.}
+#' \item{other_spp_bw}{A named list mapping other species internal column names to human-readable names.}
+#' \item{water_quality_bw}{A named list mapping water quality reading type internal column}
+#' \item{plot_palette}{A vector of colors used for plotting water quality data.}
+#' \item{plot_breaks}{A named vector mapping Riverfly species internal column names to human-readable names.}
+#' }
+riverfly_spp_bw <- list(
```

Comment:
Just checking on the capitalisation here. Common names should be lower case and the scientific names should be capitalised. With something like 
>"caseless_caddisfly" = "Caseless caddisfly (Trichoptera)",

Is "Caseless" capitalised because of how it will be used in output, for example the first word of a sentence or list?